draft-ietf-moq-transport.md

```diff
@@ -442,6 +442,28 @@ constrain the information in these fields, for example by restricting them to
 UTF-8. Any specification that does needs to specify the canonicalization into
 the bytes in the Track Namespace or Track Name such that exact comparison works.
 
+### Subscription Identifiers {#subscription-id}
+
+To optimize wire efficiency, some MoQT messages refer to a track by a numeric
+identifier, rather than the Full Track Name. When a subscriber makes a SUBSCRIBE
+or FETCH request for a track, it associates a Request ID with the request.  The
+corresponding Subscription Request ID is equal to `Request ID << 1`.
+
+MoQT also allows the publisher to choose an identifier for each track, called
+the Publisher-Chosen Subscription ID (PSID). PSIDs are odd and have a maximum
+value of 2^61-1. The same identifier MUST NOT be assigned multiple tracks in the
+same session.  PSIDs are required when the publisher initiates the subscription
+(TODO: delete this; PUBLISH message is forthcoming) and are optional when the
+subscriber initiates it.
+
+When a publisher sends Subgroups or Datagrams, these can contain either a
+Subscription Request ID or Publisher-Chosen Subscription ID, determined by the
+least significant bit of the identifier. Control messages for an established
```

Comment:
Control message needs to have both exchanged. If not, the subscriber will not be able to know which PSID corresponds to which track.
Say if Control message used RSID (8) for TrackA and object messages uses PSID (33) .. How can the subscriber know 33 is for TrackA ?

Reply:
The word `established` indicates that the subscriber has seen SUBSCRIBE_OK.